java/src/jmri/jmrix/AbstractSerialPortController.java

```diff
@@ -337,7 +337,24 @@ final protected FlowControl getFlowControl(com.fazecast.jSerialComm.SerialPort s
     
         return lastFlowControl;
     }
-    
+
+    /**
+     * Add a data listener to the specified port
+     * @param serialPort Port to be updated
+     * @param serialPortDataListener the listener to add
+     */
+    final protected void setDataListener(com.fazecast.jSerialComm.SerialPort serialPort,com.fazecast.jSerialComm.SerialPortDataListener serialPortDataListener){
+        currentSerialPort.addDataListener(serialPortDataListener);
+    }
+
+    /**
+     * Add a data listener to the specified port
+     * @param serialPort Port to be updated
+     */
+    final protected void closeSerialPort(com.fazecast.jSerialComm.SerialPort serialPort){
```

Comment:
The Javadoc for this method seems to be a copy/paste error.

Reply:
fixed in PR #12795